inspirehep/modules/disambiguation/api.py

```diff
@@ -77,3 +78,13 @@ def train_and_save_ethnicity_model():
     estimator.load_data(current_app.config['DISAMBIGUATION_ETHNICITY_DATA_PATH'])
     estimator.fit()
     estimator.save_model(current_app.config['DISAMBIGUATION_ETHNICITY_MODEL_PATH'])
+
+
+def train_and_save_distance_estimator():
+    """Train the distance estimator model and save it to disk."""
+    estimator = DistanceEstimator(current_app.config['DISAMBIGUATION_DISTANCE_MODEL_PATH'])
```

Comment:
This doesn't sound correct, given https://github.com/inspirehep/inspire-next/pull/3455/files#diff-4164879ce92c4a6f3a961ac28e1b084aR110 !

Reply:
Yup, silly mistake there.